Automattic/wp-calypso
# Sites Dashboard: Remove layout-related overrides from Dotcom and A4A

Related to https://github.com/Automattic/dotcom-forge/issues/10139

## Proposed Changes

This PR removes layout-related overrides from Dotcom and A4A. 

For details of individual overrides, please refer to this post's  pfYzsZ-15D-p2 "Style Overrides Removal" and "Remaining Style Overrides" sections. 

## Why are these changes being made?


https://github.com/Automattic/dotcom-forge/issues/10139

## Testing Instructions



On both Calypso and A4A Live;

* Go to /sites
* Click any site
* Ensure nothing breaks

A4A Live

* Go to /overview
* Go to /marketplace/assign-license
	* This page has a bit different layout from other A4A pages.
* Ensure nothing breaks

## Pre-merge Checklist
